<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph_SimilarityClass</class>
 <widget class="QMainWindow" name="Graph_SimilarityClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>735</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>735</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph_Similarity</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QPushButton" name="check_library_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查看模式库</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>33</y>
      <width>580</width>
      <height>190</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>580</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>564</width>
      <height>562</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">margin-bottom:10px;font-size:15pt;</string>
       </property>
       <property name="text">
        <string>输入需匹配元件的编码及图片</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="target_image_Label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="target_image_Button">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>选取图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>31</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font-size:14pt;</string>
             </property>
             <property name="text">
              <string>主视图特征编码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="target_front_LineEdit">
             <property name="minimumSize">
              <size>
               <width>290</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>290</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font-size:14pt;</string>
             </property>
             <property name="text">
              <string>俯视图特征编码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="target_top_LineEdit">
             <property name="minimumSize">
              <size>
               <width>290</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>290</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font-size:14pt;</string>
             </property>
             <property name="text">
              <string>侧视图特征编码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="target_side_LineEdit">
             <property name="minimumSize">
              <size>
               <width>290</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>290</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="match_Button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">margin-top:5px;margin-bottom:5px;font-size:13pt;margin-left:150px;margin-right:150px;</string>
       </property>
       <property name="text">
        <string>开始匹配</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:12pt;</string>
           </property>
           <property name="text">
            <string>匹配结果:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="match_Result_Label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:1px solid;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>主视图特征编码:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>俯视图特征编码:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>侧视图特征编码:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>120</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>主视图相似类型:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>俯视图相似类型:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>俯视图相似类型:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="front_view_type">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:13pt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="top_view_type">
           <property name="styleSheet">
            <string notr="true">font-size:13pt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="side_view_type">
           <property name="styleSheet">
            <string notr="true">font-size:13pt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>相似度:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>相似度:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="styleSheet">
            <string notr="true">font-size:14pt;</string>
           </property>
           <property name="text">
            <string>相似度:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">font-size:14pt;margin-left:115px;</string>
       </property>
       <property name="text">
        <string>三视图最终相似度:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>check_library_Button</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Graph_Similarity.qrc"/>
 </resources>
 <connections/>
</ui>
